src/core_plugins/i18n/server/__tests__/i18n.js

```diff
@@ -0,0 +1,158 @@
+import expect from 'expect.js';
+import i18n from '../i18n/i18n';
+import path from 'path';
+import Promise from 'bluebird';
+
+const PATH_SEPARATOR = path.sep;
+const DATA_PATH = __dirname + PATH_SEPARATOR + 'data';
+
+describe('Test registering translations for test_plugin_1', function () {
```

Comment:
I have re-arranged the test into a more hierarchical structure. The other methods were being implicitly called but I have updated them to have specific tests now.